### PR TITLE
For #15471: Show Delete button in red in multi-select overflow menu

### PR DESCRIPTION
**Pull request by joc-a**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

Bookmarks:
Bookmarks

History:
History

### CODE REVIEW

**Review by ekager — CHANGES_REQUESTED** [review]
I think we could use an already-created extension method we have here: see `SpannableString.kt`, we have a method `SpannableString.setTextColor` - would that work?

File: app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt [hunk 1]
```diff
@@ -44,12 +45,7 @@ import org.mozilla.fenix.R
 import org.mozilla.fenix.components.FenixSnackbar
 import org.mozilla.fenix.components.StoreProvider
 import org.mozilla.fenix.components.metrics.Event
-import org.mozilla.fenix.ext.bookmarkStorage
```
Comment (ekager) [inline app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt:47]: One more nit and then we're there! Our lint checks don't allow wildcard imports so these all need to be declared separately. The IDE probably auto changed this to try to be smart :/ but you can change the Android Studio settings for this in Preferences -> Editor -> Code style -> Kotlin and then select "Use single name import" 👍

**Review by ekager — CHANGES_REQUESTED** [review]
One lint thing to fix and then we're good!

File: app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt (hunk 1 above)
Comment (joc-a) [reply app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt:47]: Single name imports it is! 😊

**Comment by ekager** [comment]
bors try

**Review by ekager — APPROVED** [review]
Looks great!

**Comment by ekager** [comment]
bors r=ekager

**Pull request merged**